### ISSUE TITLE
CELA P1_A11y_code.visualstudio.com_WCAG2.1_Homepage_Zoom: When the page is zoomed to 400% there are 2 scroll bars available on the page. One vertical and another horizontal bar.

### ISSUE THREAD

**Issue by khansahab15** [microsoft/vscode-docs#3432]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-shbagl%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847575447&sdata=XQqvaO2rFh7p/7F2DmzFj2wgJSiYHGuJV73Xg8hPVbA%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11ySev1; #A11yTCS; #A11yWCAG2.1; #Benchmark; #CELA P1; #ChromiumEdge; #Zoom; #DesktopWeb; #MAS1.4.10; #Win10; #Visual Studio Code; #CELA P1_A11y_code.visualstudio.com_WCAG2.1`

**Environment Details**:
Application Name: code.visualstudio.com
#URL: https://code.visualstudio.com/docs/editor/integrated-terminal
Windows Version: Windows10
Browser version: Microsoft Edge 80.0.361.5

**Repro Steps:** 
1.	Launch the application URL: https://code.visualstudio.com/docs/editor/integrated-terminal
2.	Home screen will open
3.	Zoom the page to 400%.

**Actual:**
When the page is zoomed to 400% there are 2 scroll bars available on the page. One vertical and another horizontal bar.

**Same issue is observed in the following page:**
1. https://code.visualstudio.com/updates/v1_30
2. https://code.visualstudio.com/updates/v1_29

**Expected:**
There should be just one scroll bar on the page. Content should be provided without loss of data and functionality and without requiring 2 scroll bars.

**User Impact:**
Low vision users won't be able to read data in a single layout.

**Attachment:**
[#3432.zip](https://github.com/microsoft/vscode-docs/files/4265393/3432.zip)

**Comment by gregvanl** [comment]
Hi @khansahab15 This seems to be the behavior for any website on Edge if the zoom level is large enough. I don't think this is specific to the VS Code website. I see two scroll bars when I look at this issue on the GitHub site.

**Comment by khansahab15** [comment]
As received input from SME regarding Reflow,  the requirement is passing here hence closing this bug

**Comment by kapilvaishna** [comment]
**GithubTags**:#Closed;
Issue does not repro for primary page and similar page in environment https://code.visualstudio.com/docs/editor/integrated-terminal , so Closing the bug

https://user-images.githubusercontent.com/124254981/216606015-fddc40c4-4f8f-4b42-8ded-161984da7e42.mp4

**Comment by 12-shweta** [comment]
#CELA_code.visualstudio.com_WCAG2.1_FEB2020;
